docs/source/02_installation.md

```diff
@@ -43,111 +44,12 @@ alias adam-shell="${ADAM_HOME}/bin/adam-shell"
 The first alias should be used for running ADAM jobs that operate locally. The latter two aliases 
 call scripts that wrap the `spark-submit` and `spark-shell` commands to set up ADAM. You'll need
 to have the Spark binaries on your system; prebuilt binaries can be downloaded from the
-[Spark website](http://spark.apache.org/downloads.html). Currently, our continuous builds default to
-[Spark 1.5.2 built against Hadoop 2.6 (CDH5)](http://d3kbcqa49mib13.cloudfront.net/spark-1.5.2-bin-hadoop2.6.tgz),
-but any more recent Spark distribution should also work.
+[Spark website](http://spark.apache.org/downloads.html). Our [continuous integration setup](
+https://amplab.cs.berkeley.edu/jenkins/job/ADAM/) builds ADAM against Spark 1.4.1, 1.5.2, 1.6.2,
+and 2.0.0, as well as Hadoop 2.3.0 and 2.6.0.
 
 Once this alias is in place, you can run ADAM by simply typing `adam-submit` at the commandline, e.g.
 
 ```bash
 $ adam-submit
 ```
```

Comment:
Would it be appropriate to add a section about installation and running via Homebrew and Linuxbrew here?